src/checkfnis.cpp

```diff
@@ -1,29 +1,44 @@
 #include "checkfnis.h"
 
 #include "iplugingame.h"
+#include "pluginsetting.h"
 #include "report.h"
-#include <scopeguard.h>

```

Comment:
I wasn't sure of the rules. scopeguard.h and questionboxmemory.h are both part of mod organizer, but not in the checkfnis repository, but the same applies to iplugingame.h and report.h